mail_borg/mailer.py

```diff
@@ -78,8 +78,8 @@ class _Logger():
     is running. Communication between these threads is provided via a
     :class:`queue.Queue` instance.
 
-    The methods in this class that take a ``string`` argument expect that
-    said argument is can be serialized to ``JSON``.
+    The methods in this class that take a ``strings`` argument expect that
```

Comment:
```suggestion
    The methods in this class that take a ``string`` argument expect that
```

Reply:
the name of the related argument in the win32log package is ``strings`` (and even the name of the argument in the microsoft documentation about log events is ``strings``)